lib/src/components/NxFileUpload/NxFileUpload.scss

```
@@ -47,6 +47,34 @@
   }
 }
 
+
+.nx-multi-file-upload {
+  width: 348px;
+
+  // need an enclosing div around NxMultiFileUpload that doesn't contain the validationError message
+  &__container {
+    $height: 220px;
+
+    min-height: $height;
+    box-sizing:border-box;
+    border: 1px solid var(--nx-swatch-indigo-90);
+    border-radius: 6px;
+
+    &__files {
+      padding: var(--nx-spacing-6x);
+      padding-bottom: 0px;
```

I never like specifying a combined property followed by immediately overriding part of it
```suggestion
      padding: var(--nx-spacing-6x) var(--nx-spacing-6x) 0 var(--nx-spacing-6x);
```